packages/webpack/src/config/base.js

```diff
@@ -361,6 +362,22 @@ export default class WebpackBaseConfig {
       plugins.push(new HardSourcePlugin(Object.assign({}, this.options.build.hardSource)))
     }
 
+    if (!this.isServer && this.loaders.ts.transpileOnly && this.options.build.useForkTsChecker) {
+      const ForkTsCheckerWebpackPlugin = tryRequire('fork-ts-checker-webpack-plugin')
+      if (ForkTsCheckerWebpackPlugin) {
+        plugins.push(new ForkTsCheckerWebpackPlugin(Object.assign({
+          vue: true,
+          tsconfig: path.resolve(this.options.srcDir, 'tsconfig.json'),
```

Comment:
nuxt-config and other configs are usually resolved from rootDir. OR it should be better to check both, like postcss config

Reply:
Alright ! Will use `rootDir` instead !